packages/client/src/runtime/core/errorRendering/ScalarValue.ts

```diff
@@ -0,0 +1,21 @@
+import { ErrorWriter } from './base'
+import { FormattedString } from './FormattedString'
+import { Value } from './Value'
+
+export class ScalarValue extends Value {
+  constructor(readonly text: string) {
+    super()
+  }
+
+  override getPrintWidth(): number {
+    return this.text.length
+  }
+
+  override write(writer: ErrorWriter): void {
+    const string = new FormattedString(this.text)
+    if (this.hasError) {
+      string.underline().setColor(writer.context.chalk.redBright)
```

Comment:
Does the color need to come from the context? Or could we more simply pass the chalk color directly?

Reply:
So, `chalk` object in context can be non-default one, with a different configuration. In particular, if `errorFormat` is anything other than `pretty` it will be a disabled instance which won't add any colors to the strings. This allows us to easily support both colored and colorless output. [Reference code](https://github.com/prisma/prisma/pull/18421/files#diff-2f65b0b20e447d3e2bd7627410b745ee3eb24f93508608fbe40740a322554c74R33-R36)